c2corg_ui/__init__.py

```diff
@@ -68,3 +73,19 @@ def _add_static_view(config, name, path):
     )
     config.add_cache_buster(path, version_cache_buster)
     CACHE_PATH.append(name)
+
+
+class NotFound():
+
+    notFound = 'c2corg_ui:templates/404.html'
+
+    def __init__(self, request):
+        self.settings = request.registry.settings
+
+    @view_config(context=HTTPInternalServerError, renderer=notFound)
+    @view_config(context=HTTPBadRequest, renderer=notFound)
+    @notfound_view_config(renderer=notFound)
+    def notfound(self):
+        return {
```

Comment:
I think for "not found" the status code should really be 404, so that search engines correctly index the site. So, maybe we should forward the status code?